lms/static/sass/discussion/lms-discussion-main.scss

```diff
@@ -0,0 +1,2 @@
+@import 'lms/static/sass/discussion/lms-discussion-main';
+@import '../theme-overrides';
```

Comment:
@giovannicimolin, wouldn't it be better to import only the overrides corresponding to the current file if not all of them are needed everywhere this is imported? I know it will increase the number of files, but the common ones can still be imported in the specific override files.

Reply:
@lgp171188 We only need to import the `theme-colors` file here. There's no overrides for the discussion. 
As for separating each override to a specific file were it is used, I won't be able to do that in the time that I have left on my onboarding budget. Can we close up this task with these modifications and create a follow up one following the discovery done on SE-720?

Reply:
@lgp171188 I've changed the override to `theme-colors` on this file.